howtos/custom_generators.rst

```diff
@@ -176,21 +101,18 @@ You ``export`` the package recipe to the local cache, so it can be used by other
 Using the generator
 -------------------
 
-Let's create a test project that uses this generator, and also an existing library conan package,
-we will use the simple "Hello World" package we already created before:
+Let's create a test project that uses this generator. We will use a simple "Hello World" program
+and also an existing library conan package, we will use the simple "Hello World" package we already created before:
```

Comment:
Where is this "Hello World" package? If I enter this page of the documentation, I haven't created anything before.

I think that every _How to_ should be self-contained.